source/_docs/configuration/templating.markdown

```diff
@@ -195,7 +195,19 @@ The same thing can also be expressed as a filter:
 - Filter `timestamp_utc` converts a UNIX timestamp to its string representation representation as date/time in UTC timezone.
 - Filter `timestamp_custom(format_string, local_time=True)` converts an UNIX timestamp to its string representation based on a custom format, the use of a local timezone is default. Supports the standard [Python time formatting options](https://docs.python.org/3/library/time.html#time.strftime).  
 
-Note: [UNIX timestamp](https://en.wikipedia.org/wiki/Unix_time) is the number of seconds that have elapsed since 00:00:00 UTC on 1 January 1970. Therefore, if used as a function's argument, it can be substituted with a numeric value (`int` or `float`):  
+<div class='note'>
+[UNIX timestamp](https://en.wikipedia.org/wiki/Unix_time) is the number of seconds that have elapsed since 00:00:00 UTC on 1 January 1970. Therefore, if used as a function's argument, it can be substituted with a numeric value (`int` or `float`):  
```

Comment:
The parser can only switch between HTML & Markdown context if there is at least an empty line between HTML & Markdown.

So, after the opening div, a new line need to be added (and before the closing div).

Same applies to the added div below.

Additionally, this paragraph ends with `:`. Instead of now showing something another big box is shown. Which is a bit weird?